Python/Product/PythonTools/PythonTools/Project/PythonProjectLaunchProperties.cs

```diff
@@ -305,6 +305,9 @@ IProjectLaunchProperties properties
             IServiceProvider provider = null
         ) {
             var pathEnv = project.GetProjectAnalyzer().InterpreterFactory.Configuration.PathEnvironmentVariable;
+            if (pathEnv == null || pathEnv == String.Empty)

```

Comment:
When I look at the UI in Tools > Options > Python Tools > Environment Options and I select this interpreter, it does say PYTHONPATH in the box.  So it may very well be a bug that `project.GetProjectAnalyzer().InterpreterFactory.Configuration.PathEnvironmentVariable` is returning String.Empty when the value has been defaulted.  Still might be nice to have this fallback just in case, though.

I don't really understand why this works for the Python-only case (e.g. Enable Native debugging not checked).  I confirmed that the VsDebugInfo was being passed through exactly the same.  It must be an implementation detail of the Native code debug engine.